### ISSUE TITLE
As a teacher I would like to have a login to this project

### ISSUE THREAD

**Issue by yeeerdley-itu** [ITU-BDSA2024-GROUP29/Chirp#60]
Issue #58, #59 & #56 should be completed before handling this issue

**Comment by yeeerdley-itu** [comment]
I have been informed that this requirement might no longer be relevant. Will need to confirm with TA later today to verify, issue is ON HOLD until then

**Comment by yeeerdley-itu** [comment]
due to inconclusive answers from our TA last week, this tasks will be marked as done. Even though we did **not** include this functionality
